Source/SourceKittenFramework/File.swift

```diff
@@ -293,17 +294,17 @@ public struct File {
     /**
     Parses `dictionary`'s documentation comment body.
 
-    :param: dictionary Dictionary to parse.
-    :param: syntaxMap  SyntaxMap for current file.
+    - parameter dictionary: Dictionary to parse.
+    - parameter syntaxMap:  SyntaxMap for current file.
     
-    :returns: `dictionary`'s documentation comment body as a string, without any documentation
-              syntax (`/** ... */` or `/// ...`).
+    - returns: `dictionary`'s documentation comment body as a string, without any documentation
+               syntax (`/** ... */` or `/// ...`).
     */
     public func getDocumentationCommentBody(dictionary: XPCDictionary, syntaxMap: SyntaxMap) -> String? {
-        return flatMap(SwiftDocKey.getOffset(dictionary)) { offset in
-            return flatMap(syntaxMap.commentRangeBeforeOffset(Int(offset))) { commentByteRange in
-                return flatMap(contents.byteRangeToNSRange(start: commentByteRange.start, length: commentByteRange.length)) { nsRange in
-                    return contents.commentBody(range: nsRange)
+        return SwiftDocKey.getOffset(dictionary).flatMap { offset in
+            return syntaxMap.commentRangeBeforeOffset(Int(offset)).flatMap { commentByteRange in
+                return contents.byteRangeToNSRange(start: commentByteRange.start, length: commentByteRange.length).flatMap { nsRange in
+                    return contents.commentBody(nsRange)
```

Comment:
return statements in a closure are necessary when the input parameters are named (`nsRange in...`). Alternatively, we could use optional binding instead:

```swift
public func getDocumentationCommentBody(dictionary: XPCDictionary, syntaxMap: SyntaxMap) -> String? {
    if let offset = SwiftDocKey.getOffset(dictionary),
        commentByteRange = syntaxMap.commentRangeBeforeOffset(Int(offset)),
        nsRange = contents.byteRangeToNSRange(start: commentByteRange.start, length: commentByteRange.length) {
        return contents.commentBody(nsRange)
    }
    return nil
}
```